apps/google-analytics-4/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -1,259 +1,10 @@
-import { useCallback, useState, useEffect } from 'react';
-import { AppExtensionSDK } from '@contentful/app-sdk';
-import { Heading, Form, Paragraph } from '@contentful/f36-components';
-import { css } from 'emotion';
-import { omitBy } from 'lodash';
-import { useSDK } from '@contentful/react-apps-toolkit';
-import tokens from '@contentful/f36-tokens';
-
-import FormControlServiceAccountKey from '../components/FormControlServiceAccountKey';
-import {
-  convertServiceAccountKeyToServiceAccountKeyId,
-  convertKeyFileToServiceAccountKey,
-  AssertionError,
-} from '../utils/serviceAccountKey';
-import type { AppInstallationParameters, ServiceAccountKey, ServiceAccountKeyId } from '../types';
-
-const googleAnalyticsBrand = {
-  primaryColor: '#E8710A',
-  url: 'https://www.google.com/analytics',
-  logoImage: './images/google-analytics-logo.png',
-};
-
-const styles = {
-  body: css({
-    height: 'auto',
-    minHeight: '65vh',
-    margin: '0 auto',
-    marginTop: tokens.spacingXl,
-    padding: `${tokens.spacingXl} ${tokens.spacing2Xl}`,
-    maxWidth: tokens.contentWidthText,
-    backgroundColor: tokens.colorWhite,
-    zIndex: 2,
-    boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
-    borderRadius: '2px',
-  }),
-  background: css({
-    display: 'block',
-    position: 'absolute',
-    zIndex: -1,
-    top: 0,
-    width: '100%',
-    height: '300px',
-    backgroundColor: googleAnalyticsBrand.primaryColor,
-  }),
-  section: css({
-    margin: `${tokens.spacingXl} 0`,
-  }),
-  splitter: css({
-    marginTop: tokens.spacingL,
-    marginBottom: tokens.spacingL,
-    border: 0,
-    height: '1px',
-    backgroundColor: tokens.gray300,
-  }),
-  sectionHeading: css({
-    fontSize: tokens.fontSizeL,
-    marginBottom: tokens.spacing2Xs,
-  }),
-  serviceAccountKeyFormControl: css({
-    marginBottom: tokens.spacing2Xl,
-  }),
-  icon: css({
-    display: 'flex',
-    justifyContent: 'center',
-    img: {
-      display: 'block',
-      width: '170px',
-      margin: `${tokens.spacingXl} 0`,
-    },
-  }),
-};
+import React from 'react'
```

Comment:
Common Pattern Comment: Treat these location Components as an App.tsx for each location app we are building out